core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableMetadata.java

```diff
@@ -83,16 +83,6 @@ public Optional<String> getComment()
         return comment;
     }
 
-    /**
-     * Engine would not violate the check constraint when predicate evaluates to UNKNOWN (e.g. NULL > 100).
-     * @return List of string representation of a Trino SQL scalar expression that can refer to table columns by name and produces a result coercible to boolean
-     */
-    @Experimental(eta = "2023-03-31")
-    public List<String> getCheckConstraints()
-    {
-        return checkConstraints;
```

Comment:
It's weird that the class accepts something in the ctor, but doesn't let users to query that information, except unless transformed to a different object.

Please add it back here

Reply:
Added it back.